tests/README.md

```diff
@@ -182,27 +138,36 @@ expected output. The framework includes the following built-in verify functions:
    1. For each line in `expected` with the prefix "- " there must be no line in
       the output containing the following string.
 
-1. `_verify_failure`** `func`
+1. **`_verify_failure`** `func`
 
    Runs `func` and confirms that it fails (i.e., non-zero return code). This function is useful
    for testing commands that demonstrate configurations that are expected to fail.
 
-## Running the Tests: Make
+After all test steps are run, add
+```sh
+#! cleanup
+```
+as a single line followed by the cleanup steps that clean up all the resources. These steps can also directly call the functions defined in `util/samples.sh` and `snips.sh`.
 
-You can execute all istio.io tests using make.
+## Running the Tests
 
+Run
 ```bash
-make test.kube.presubmit
+make doc.test
 ```
-
-Alternatively, you can run the tests in a particular package under `tests/`.
-For example, the following command will only run the traffic management tests:
-
+to start testing all docs in the content folder within a `kube` environment. This command takes two optional environment variables. One is `ENV` that specifies the test environment (either `native` or `kube`), and is `kube` by default. The other is `TEST` that specifies the tests to be run using the name of the directory. For example, the command
+```bash
+make doc.test TEST=traffic-management
+```
+will run all the tests under `traffic-management` folder. The `TEST` variable also accepts multiple test names separated by commas, for example,
```

Comment:
Now that the tests are in the content tree, there may be more folders with the same name (e.g., under /docs/tasks and under /docs/ops), so you will need to qualify the names (e.g., TEST=tasks/traffic-management)

Reply:
Sounds good! I'll document that. I didn't see snips.sh files under docs/ops so I thought it might not be a problem as the framework would only find `test.sh` files